README.md

```diff
@@ -1,4 +1,41 @@
-# GT Scheduler
-> Georgia Tech Scheduler lets you find the schedule that fits you best among all the possible combinations of courses.
+# GT Scheduler V2.0
 
-[![Screenshot](https://jasonpark.me/gt-scheduler/screenshot.png)](https://jasonpark.me/gt-scheduler/)
+GT (Georgia Tech) Scheduler Version 2.0 is a comprehensive platform for Semester Planning and Course Scheduling!
+
+With GT Scheduler V2.0 you get immediate access to course critique evaluations, professor information and data, and more!
+
+## What's New
+
+In Version 2.0, users have access to many more features:
+
+### 1. Course Critique Integration
+
+Click the info Icon on the right side of the course to get detailed grade information from critique.gatech.edu
+
+View detailed graphs showing **grade distributions and comparisons** among different professors and the course averages!
+
+After selecting your sections, take a look at the **Cummulative Average GPA**!
+
+### 2. Dark Mode Support
+
+Use the button on the bottom of the page to toggle **Dark** and **Light** Modes!
+
+### 3. UI and UX Improvements
+
+Resize-able and Collapse-able Columns, Material Design in both Light Mode and Dark Mode, Animated UI Elements for a better User Experience!
+
+## Coming Soon
+
+I plan to have many more features coming soon!
+
+- Viewing and Tracking your Degree Requirements by major
+- Add "No Class" Time Blocks
+- Comprehensive Course Difficulty
+- More Graphs and Data Visualizations
+- More UI improvements!
+
+![GT Scheduler Split Theme](https://raw.githubusercontent.com/abhitirumala/gt-scheduler-v2/master/public/image.jpg)
+
+## Credits
+
+Credits to **Jason Park** (https://github.com/64json/gt-scheduler) for initially developing the popular web application and helping the Georgia Tech Community in making course registration an easier task.
```

Comment:
Revert this as well. If you wish to include change log, do so in `CHANGELOG.md`. For the coming-soon-features, we can create a GitHub issue for each of them to have a discussion thread.